src/poggit/release/submit/PluginSubmission.php

```diff
@@ -332,7 +332,7 @@ public function processArtifact() {
         $py["version"] = $this->version;
         $py["api"] = SubmitFormAjax::rangesToApis($this->spoons);
         file_put_contents($pharUrl . "plugin.yml", yaml_emit($py));
-        if((!is_file($pharUrl . "LICENSE") || !is_file($pharUrl . "LICENSE.md") || !is_file($pharUrl . "LICENSE.MD")) && $this->license->type !== null && $this->license->type !== "custom" && $this->license->type !== "none") {
+        if((!is_file($pharUrl . "LICENSE") || !is_file($pharUrl . "LICENSE.md") || !is_file($pharUrl . "LICENSE.MD") || !is_file($pharUrl . "LICENSE.txt")) && $this->license->type !== null && $this->license->type !== "custom" && $this->license->type !== "none") {
```

Comment:
does github recognize the file with a lowercase name?

Reply:
This parts GitHub irrelevant, but I don't know, I think so.

Reply:
I think we should probably accept lowercase filenames.

Reply:
[58375f6](https://github.com/poggit/poggit/pull/325/commits/58375f61b23ec4af46e85abb0e5fbce472601a46)